src/main/java/org/cactoos/func/FuncOf.java

```diff
@@ -95,7 +95,7 @@ public FuncOf(final Proc<X> proc, final Y result) {
      * Ctor.
      * @param fnc Func
      */
-    public FuncOf(final Func<X, Y> fnc) {
+    private FuncOf(final Func<X, Y> fnc) {
```

Comment:
@memoyil Sorry, but I believe private constructors or methods are a code smells. Why? Because they can hide functionality and decrease reusability.